src/main/java/edu/kis/powp/jobs2d/drivers/proxy/DeviceUsageProxy.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.powp.jobs2d.drivers.proxy;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.manager.DeviceUsageManager;
+
+
+public class DeviceUsageProxy implements Job2dDriver {
```

Comment:
In the GoF design patterns catalog such design is called 'decorator' not 'proxy' (proxy wraps a concrete class).

Reply:
renamed